src/components/Form.js

```diff
@@ -154,12 +155,15 @@ class Form extends React.Component {
                     this.setTouchedInput(inputID);
                     this.validate(this.inputValues);
                 },
-                onChange: (value) => {
+                onChangeText: (value) => {
```

Comment:
@parasharrajat do you think `onInputChange` is a better name? Because `Picker`, `Checkboxes` etc are also part of the form.

Reply:
Good call.